src/s1reader/s1_reader.py

```diff
@@ -248,6 +249,35 @@ def get_burst_centers_and_boundaries(tree):
 
     return center_pts, boundary_pts
 
+def get_ipf_version(tree:ET):
+    '''Extract the IPF version from the ET of manifest.safe
+    '''
+    elem_metadata_section=tree.find('metadataSection')
+    flag_found_ipf_version=False
+    idx_processing=0
+    for idx_processing,elem_sub in enumerate(elem_metadata_section):
+        if elem_sub.attrib['ID']=='processing':
+            flag_found_ipf_version=True
+            break
+
+    if flag_found_ipf_version:
+        str_ipf_version=elem_metadata_section[idx_processing][0][0][0][0][0].attrib['version']
+        return float(str_ipf_version)
+
+def is_eap_correction_necesasry(ipf_version:float) -> int :
+    '''Examines if what level of EAP correction is necessary, based on the IPF version
+    0: No EAP correction necessary (i.e. correction already applied)
+    1: Phase-only correction is necessary
+    2: Phase and Magniture correction is necessary'''
+    ipf_ver_int=int(ipf_version*100)
+    #Based on ESA technical document
+    if ipf_ver_int>=243:
+        return 0 # No EAP correction necessary (i.e. correction already applied)
+    elif ipf_ver_int>=236:
+        return 1 # Phase-only correction is necessary
+    else:
+        return 2 # Phase and Magniture correction is necessary
```

Comment:
```suggestion
    eap = SimpleNamespace()
    eap.phase_correction = True in ipf_ver_int < 243 else False
    eap.magnitude_correction = True in ipf_ver_int < 236 else False
    
    return eap
```
What do you think about something discrete and self documenting like a `SimpleNamespace`?